### PR TITLE
WIP: Section on asm scanning changed

### PR DESCRIPTION
**Pull request by SeanFeldman**
@andreasohlund IIRC you've mentioned that there's a way for existing customers that rely on nested folders asm scanning to have those scanned using profiles. Could you please add that info to the this PR?

### CODE REVIEW

File: nservicebus/upgrades/5to6.md [hunk 1]
```diff
@@ -168,3 +168,12 @@ This is no longer supported. We advise to, instead of using `IBuilder` directly,
 ## AutoSubscribe
 
 The configuration option `DoNotRequireExplicitRouting()` has been obsoleted since transports with support for centralized pubsub will always auto subscribe all events without requiring explicit routing. Transports with message driven pubsub (like MSMQ, SqlServer and AzureStorageQueues) will not subscribe properly if there is no routing specified. If you used this option you can safely be removed it.
+
+## Assembly scanning
+
+Default behaviour for assembly scanning has changed and NServiceBus is no longer scanning assemblies in nested folders.
```
Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: > NServiceBus is no longer scanning assemblies in nested folders

u need to call out "since which version" here

Comment (indualagarsamy) [inline nservicebus/upgrades/5to6.md]: I'd also add a note to the user what they need to do, if they need to scan the nested folder in the new version.

Comment (SeanFeldman) [inline nservicebus/upgrades/5to6.md]: @indualagarsamy I will once have the info. See here: https://github.com/Particular/docs.particular.net/pull/764#issue-102252535

**Comment by SeanFeldman** [comment]
@andreasohlund do you recall conversation about nested folders, asm scanning and need for custom profiles?

**Comment by andreasohlund** [comment]
Can't remember, there doesn't seem to be any host specific setting to
control the folders. But since you can explicitly tell the host to use a
given endpoint config type you can don't really need to turn the subfolder
scanning on? (since the actual endpoint will rescan anyway)

http://docs.particular.net/nservicebus/hosting/nservicebus-host/#configuring-your-endpoint

**Comment by SimonCropp** [comment]
@SeanFeldman what do u need to get this finished?

**Comment by SeanFeldman** [comment]
Will add usage comparison (whitelist vs blacklist).

**Pull request merged**